extension/remotetapextension/README.md

```diff
@@ -0,0 +1,31 @@
+# Remote Tap Extension
+
+<!-- status autogenerated section -->
+| Status        |           |
+| ------------- |-----------|
+| Stability     | [development]  |
+| Distributions | [] |
+| Issues        | [![Open issues](https://img.shields.io/github/issues-search/open-telemetry/opentelemetry-collector-contrib?query=is%3Aissue%20is%3Aopen%20label%3Aextension%2Fremotetap%20&label=open&color=orange&logo=opentelemetry)](https://github.com/open-telemetry/opentelemetry-collector-contrib/issues?q=is%3Aopen+is%3Aissue+label%3Aextension%2Fremotetap) [![Closed issues](https://img.shields.io/github/issues-search/open-telemetry/opentelemetry-collector-contrib?query=is%3Aissue%20is%3Aclosed%20label%3Aextension%2Fremotetap%20&label=closed&color=blue&logo=opentelemetry)](https://github.com/open-telemetry/opentelemetry-collector-contrib/issues?q=is%3Aclosed+is%3Aissue+label%3Aextension%2Fremotetap) |
+| [Code Owners](https://github.com/open-telemetry/opentelemetry-collector-contrib/blob/main/CONTRIBUTING.md#becoming-a-code-owner)    | [@atoulme](https://www.github.com/atoulme) |
+
+[development]: https://github.com/open-telemetry/opentelemetry-collector#development
+<!-- end autogenerated section -->
+
+This extension runs as a Web server that loads the remote observers that are registered against it.
```

Comment:
> loads the remote observers

what does this mean? Is it loading executable code? Could this link to more details about "remote observers"

Reply:
See here: https://github.com/open-telemetry/opentelemetry-collector-contrib/tree/main/processor/remoteobserverprocessor

Reply:
Sorry, that doesn't help. WebSocket processor just streams data out, it doesn't have any observers that "register".

Reply:
I believe the intent of the description is:

> This extensions connections to a configured websocket processor to provide a simple web interface for examining running data.

Please correct me if I am wrong here.

Reply:
Websocket processor was renamed to remote observer processor. The folder and code was changed but the readme lags behind, sorry.